.gitignore

```diff
@@ -18,6 +18,8 @@ _build
 build
 .venv
 .vscode
+__pycache__
+generated_script
```

Comment:
Don''t add it here. Add it to the gitignore file inside the ``generated_scripts`` folder.